shared/helpers.ts

```diff
@@ -66,7 +65,7 @@ const getInstalledVersionInfo: (resource: 'node' | 'firefox' | 'sdk') => {
   const pointPath = getPointPath()
   try {
     return JSON.parse(
-      fs.readFileSync(path.join(pointPath, `${file}.json`)).toString()
+      (await fs.readFile(path.join(pointPath, `${file}.json`))).toString()
```

Comment:
I'm actually pretty confused. We're making functions `async` from sync and yet we're `await`ing them, which yields sync behavior only. What's the point then?

Reply:
`await` doesn't convert async function to sync, it's just syntactic sugar and is equivalent to `then()`.
And it's a huge difference, sync calls block the main thread, while async don't. Actually, you should always avoid sync calls to fs, it makes the whole app much slower.

Reply:
I read a couple of threads on Stack Overflow and totally agree with you here. Sync calls block the entire thread, so any other tasks running too will stop, while `await` only waits for the `await`ed function to return before proceeding further, while other tasks can continue running. PR approved from my side.